opentelemetry-example-app/src/opentelemetry_example_app/metrics_example.py

```diff
@@ -11,26 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+"""
+This module serves as an example for a simple application using metrics
+"""
 
-# pylint: skip-file
 from opentelemetry import metrics
+from opentelemetry.sdk.metrics import Meter
 
-METER = metrics.Meter()
-MEASURE = metrics.create_measure(
-    "idle_cpu_percentage",
-    "cpu idle over time",
-    "percentage",
-    metrics.ValueType.FLOAT,
-    ["environment"],
+metrics.set_preferred_meter_implementation(lambda _: Meter())
+meter = metrics.meter()
+counter = meter.create_metric(
+    "sum numbers",
+    "sum numbers over time",
+    "number",
+    int,
+    metrics.MetricKind.COUNTER,
+    ("environment"),
 )
 
-# Metrics sent to some exporter
-MEASURE_METRIC_TESTING = MEASURE.get_handle("Testing")
-MEASURE_METRIC_STAGING = MEASURE.get_handle("Staging")
+label_values = "staging"
+
+counter_handle = counter.get_handle(label_values)
```

Comment:
This example would be especially helpful if it covered all three ways of recording metrics -- via `Meter.record_batch`, via the instrument, and then via handles. This would certainly be an easier on-ramp to using metrics than reading the spec.

Reply:
That's a good point. Recording via the instrument has not been implemented yet, as this needs label sets to be implemented. I will definitely be including that in the PR for label sets.